libpod/events/logfile.go

```diff
@@ -59,7 +60,9 @@ func (e EventLogFile) Read(ctx context.Context, options ReadOptions) error {
 		}
 		go func() {
 			time.Sleep(time.Until(untilTime))
-			t.Stop()
+			if err := t.Stop(); err != nil {
+				logrus.Errorf("Stopping logger: %v", err)
```

Comment:
Is this a potential race condition where the container is already stopped and/or deleted?

Reply:
I don't think so. `t` is the `Tail` to read the logs, not the container.